ovos_core/intent_services/__init__.py

```diff
@@ -392,24 +435,8 @@ def handle_get_intent(self, message):
         utterance = message.data["utterance"]
         lang = get_message_lang(message)
 
-        # Create matchers
-        padatious_matcher = PadatiousMatcher(self.padatious_service)
-
-        # List of functions to use to match the utterance with intent.
-        # These are listed in priority order.
-        # TODO once we have a mechanism for checking if a fallback will
-        #  trigger without actually triggering it, those should be added here
-        match_funcs = [
-            padatious_matcher.match_high,
-            self.adapt_service.match_intent,
-            # self.fallback.high_prio,
-            padatious_matcher.match_medium,
-            # self.fallback.medium_prio,
-            padatious_matcher.match_low,
-            # self.fallback.low_prio
-        ]
         # Loop through the matching functions until a match is found.
-        for match_func in match_funcs:
+        for match_func in self.get_pipeline(skip_converse=True, skip_fallback=True):
```

Comment:
should the skips be hard-coded or should it be a set/tuple/list of matchers to skip?

Reply:
these are the special cases that should be skipped when getting registered intents via bus api, just added these for backwards compat but this whole api needs some re thinking
- converse / fallback / common qa need to be accounted for
- need to be made to report if they will handle an intent, without doing so, which doesnt play nice with any existing skills

i think for now we are kinda stuck with hardcoding these 2 until they work better, hopefully no more stages willl be added to the pipeline that behave like these 2